script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -159,7 +159,7 @@ Some students even call her the real student council president, because she is m
 It goes without saying that when I ran into trouble last week, she helped me out.
 <Page43>
 ...It's actually a bit of a blur.
-I'm not sure if it's because I'm not a morning person, or because my head got foggy from the thought of going back to that home from seven years ago.
+As Senpai said, I'm not sure if it's because I'm not a morning person, or because my head got foggy from the thought of going back to that home from seven years ago.//先輩の言う通り、朝に弱いことプラス、七年ぶりの帰宅で頭が曇っていたとしか思えない。
 I...
 C:>Sadly, there's nothing to talk to her about.
 C:>I have a few questions.
```

Comment:
I feel like we fixed this once, maybe it got lost. but "I... Sadly there's nothing to talk about" and "I... I have a few questions" are both awkward formulations.

maybe 
> I...
> ...have nothing to talk to her about, sadly.
> ...have a few questions.

and if we lead with an ellipsis on the choice, put a space between the `C:>` and the `...`

Reply:
It *has* been modified, just not on main or on my branch. It's been modified in Salt-Shaker's-revisions/Day 1 Edits